stack.yaml

```diff
@@ -4,7 +4,7 @@ packages:
 - .
 
 docker:
-  enable: true
+  enable: false
```

Comment:
why this change? we want to build inside docker to have the same linux installation as AWS Lambda

Reply:
"make build" has "--docker" so that builds in docker but i also want "make buildc" for local development. i can go either way on this--so i can revert if you don't find it necessary.

Reply:
Let's do a quick huddle on this on slack. I want to figure out what would be "least astonishing" so having feedback from both of you would be helpful.